libminifi/test/resources/TestHTTPGet.yml

```diff
@@ -19,6 +19,7 @@
 Flow Controller:
     name: MiNiFi Flow
     id: 2438e3c8-015a-1000-79ca-83af40ec1990
+    version: 1
```

Comment:
Isn't this the flow config yaml version and not the flow version ? Is the version above intended to be the config yaml version or the flow version?

Reply:
@phrocker that's flow config yml version, i thought it refer to the the flow version. when you update the YML file, flow version change also